langchain_dev.py

```diff
@@ -0,0 +1,171 @@
+import os
+import json
+from diarize import get_token
+from pathlib import Path
+from typing import Any
+from langchain_community.llms import HuggingFaceEndpoint
+from langchain.chains import LLMChain
+from langchain.prompts import PromptTemplate
+from langchain import FewShotPromptTemplate
+from langchain_community.document_loaders import WebBaseLoader
+from langchain_community.document_loaders import TextLoader
+from langchain_community.embeddings.sentence_transformer import (
+    SentenceTransformerEmbeddings,
+)
+from langchain_community.vectorstores import Chroma
+from langchain_text_splitters import TokenTextSplitter, CharacterTextSplitter, RecursiveCharacterTextSplitter
+from langchain import hub
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.runnables import RunnablePassthrough
+from langchain.chains import RetrievalQA
+from langchain.memory import ConversationBufferMemory
+from langchain.chains import ConversationalRetrievalChain
+
+
+os.environ["HUGGINGFACEHUB_API_TOKEN"] = get_token()
+
+def llm_init(repo_id: str = "google/gemma-7b") -> Any:
```

Comment:
Should probably not default to gemma, it's gated so we'll need to submit a request to use it. We can use it regardless, but the default should be something else that isn't gated.